CHANGES.rst

```diff
@@ -4,6 +4,10 @@ Changelog
 5.1.2 (unreleased)
 ------------------
 
+- Disallow passing strings as addresses to ``ClientStorage`` under
```

Comment:
The main point of this isn't "strings". The point is that strings are used for unix-domain sockets, which aren't supported on windows.

Please reword, along the lines of:

- Disallow passing strings as addresses to ``ClientStorage`` under Windows because string addresses are used for unix-domain sockets, which aren't supported on Windows.

Reply:
OK, sure, that makes sense :) I was a little fast and loose with the change notes because I knew that there would be merge conflicts and they would probably need reworked anyway between #110 #112 and #111 . It's just down to which one gets merged first.